canonical-web-and-design/cookie-policy
# Remove padding from the close button

## Done
Removed the padding from the close button. Performed a few quick drive by fixes.

## QA
- Run `gulp build`
- Open index.html
- Check the cookie notice look as it did before
